src/script/auth/component/ClientItem.tsx

```diff
@@ -62,18 +62,24 @@ const ClientItem = ({selected, onClientRemoval, onClick, client, clientError, re
   const [password, setPassword] = useState('');
   const [isValidPassword, setIsValidPassword] = useState(true);
   const [validationError, setValidationError] = useState<ValidationError | null>(null);
+  const [isOpen, setIsOpen] = useState(requirePassword && (isSelected || isAnimating));
```

Comment:
Looks so obvious when I see your solution. Couldn't figure one out when I looked at it, props!